### PR TITLE
Please review the code and let me know what you think!

### PR DESCRIPTION
**Pull request by ramaneswaran**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from bot.tele_bot import TeleBot 
+
+#Bot Token
+bot_token = '1006827845:AAGhqvI__6q7BYGOUwlKYLHlLpXaO-SoUYI'
+
```
Comment (Geek-ubaid) [inline src/main.py:5]: Push this to .env file and fetch all the secrets from the file . Invalidate this token and create a new one.

File: src/utils/detector.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+def identify(text):
+    keywords = ['meeting', 'party']
+    for keyword in keywords:
+        if keyword in text:
```
Comment (Geek-ubaid) [inline src/utils/detector.py:4]: It's not necessary we use word meeting and party other words may also be there. So make sure you have some context driven model here for identifying the context of the messages and then classify it as a meeting message or party etc. depends on the type of categories you are considering.

File: src/utils/parser.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    return [ent for ent in ents if ent.label_ == 'TIME']
+
+def date_parser(ents):
+    return [ent for ent in ents if ent.label_ == 'DATE']
+
+def location_parser(message):
+    
+    
+    places_list = ['SJT','TT','SMV'] #List of possible location meetings
+
```
Comment (Geek-ubaid) [inline src/utils/parser.py:20]: Don't hardcode values, we can have meeting anywhere if you are considering VIT. Include NER for location here for detecting location and time from messages.

**Review by Geek-ubaid — CHANGES_REQUESTED** [review]

File: src/utils/detector.py (hunk 2 above)
Comment (ramaneswaran) [reply src/utils/detector.py:4]: Yes, I will look into this . I wanted to make a model but didn't know how to create training data.

File: src/main.py (hunk 1 above)
Comment (ramaneswaran) [reply src/main.py:5]: Yep, rookie mistake, I will put a new token in .env

File: src/utils/parser.py (hunk 3 above)
Comment (ramaneswaran) [reply src/utils/parser.py:20]: The NER for location performs very poorly, the Time and Date NER works well, thats why I left out the NER for location. Moreover I think having a list of locations might help users to customize this for their specific use

File: src/utils/detector.py (hunk 2 above)
Comment (Geek-ubaid) [reply src/utils/detector.py:4]: Do the same so that I can merge it.

File: src/main.py (hunk 1 above)
Comment (Geek-ubaid) [reply src/main.py:5]: Create new env file and push it so that I cna merge the PR

File: src/utils/detector.py (hunk 2 above)
Comment (Geek-ubaid) [reply src/utils/detector.py:4]: You will be having some messages as your training data.

File: src/utils/parser.py (hunk 3 above)
Comment (Geek-ubaid) [reply src/utils/parser.py:20]: Use RASA's context model here for identifying the entities like location date and time. You can also try spacy models for the same. If you go with specific locations , we need to consider other locations which are outside vit.

**Comment by Geek-ubaid** [comment]
@ramaneswaran do the changes requested and push the latest commit

**Pull request closed without merging**